src/components/dashboard/Amount.js

```diff
@@ -76,26 +84,23 @@ const Amount = (props: AmountProps) => {
       return true
     }
 
-    log.info('canContiniue?', { weiAmount, params })
+    log.info('canContiniue?', { weiAmount, balance, params })
 
     try {
-      const fee = await goodWallet.calculateTxFee(weiAmount)
-      const amount = new BN(weiAmount)
-      const amountWithFee = amount.add(fee)
-      const canSend = amountWithFee.lte(new BN(balance))
-
```

Comment:
what happened to that?

Reply:
it was dup of the wallet method, removed & replaced with canSend (or canSendNative correspondingly)